packages/types/src/modeOptions.ts

```diff
@@ -29,7 +29,6 @@ export interface RunModeOptions extends CommonModeOptions {
 export type TestingType = 'e2e' | 'component'
 
 export interface OpenModeOptions extends CommonModeOptions {
-  _?: (string | null)[] | null
```

Comment:
this is a real property, it contains argv that did not match a known option. but since it's unused doesn't matter here